state-chain/chains/src/sol/sol_tx_core/tests.rs

```diff
@@ -40,6 +40,102 @@ enum BankInstruction {
 	Withdraw { lamports: u64 },
 }
 
+#[cfg(test)]
+mod versioned_transaction {
+	use sol_prim::{
+		consts::{const_address, const_hash},
+		AddressLookupTableAccount,
+	};
+
+	use crate::sol::sol_tx_core::transaction::{
+		v0::VersionedMessageV0, VersionedMessage, VersionedTransaction,
+	};
+
+	use super::*;
+
+	#[test]
+	fn create_simple_tx() {
```

Comment:
What is this testing? There aren't any assertions. Just checking we don't panic?

Reply:
Removed